scalajslib/worker/1/src/ScalaJSWorkerImpl.scala

```diff
@@ -33,10 +34,11 @@ class ScalaJSWorkerImpl extends ScalaJSWorkerApi {
       fullOpt: Boolean,
       moduleKind: ModuleKind,
       esFeatures: ESFeatures,
+      moduleSplitStyle: ModuleSplitStyle,
       dest: File
   )
-  private def minorIsGreaterThanOrEqual(number: Int) = ScalaJSVersions.binaryEmitted match {
```

Comment:
`binaryEmitted` for Scala.js `1.10` is `1.8` for application level forward binary compatibility.
We want to know the build Scala.js version which can be accessed in current and looks like `1.10.0`